google-cloud-bigquery/src/test/java/com/google/cloud/bigquery/BigQueryImplTest.java

```diff
@@ -2863,4 +2865,27 @@ public void testTestIamPermissionsWhenNoPermissionsGranted() {
     assertEquals(perms, ImmutableList.of());
     verify(bigqueryRpcMock).testIamPermissions(resourceId, checkedPermissions, EMPTY_RPC_OPTIONS);
   }
+
+  @Test
+  public void testSumeet() throws Exception {
```

Comment:
Please clean up personal tests.

Reply:
Removed the personal test(s).
Thanks for pointing it out.

Please re-run the checks 😄